realm/realm-annotations-processor/src/test/resources/some/test/RealmSetModel.java

```diff
@@ -30,6 +34,12 @@
     private RealmSet<Byte> byteSet;
     private RealmSet<byte[]> binarySet;
     private RealmSet<ObjectId> objectIdSet;
+    private RealmSet<UUID> uuidSet;
+    private RealmSet<Float> floatSet;
+    private RealmSet<Double> doubleSet;
+    private RealmSet<Boolean> booleanSet;
+    private RealmSet<Date> dateSet;
+    private RealmSet<Decimal128> decimal128Set;
```

Comment:
Is there a test failing when the model is not coverting all the various set-types? At least I didn't catch that we didn't include this in each of the typed PRs. Maybe not required at all when you have test for the various types at library level 🤔. 
But, as previously pointed out it would be nice if we had a generated proxy as part of the repository, as it is much easier to reason about generator changes when seeing the output. Maybe not for all the types, but at least one.

Reply:
> Is there a test failing when the model is not coverting all the various set-types?

That's right. I have added a `RealmSetModelWrongType` to test it.

> if we had a generated proxy

It is hard now to get a meaningful proxy since there are still a few methods missing or they need refinement (e.g. `copyToRealmOrUpdate`, `copyFromRealm`). The problem also with a working proxy is that the only sort of "proper test" we do for proxy validation is with `AllTypes` but I can't put set or dictionary fields in that class since we don't support `insert` and `createFromJson` methods yet, and we use that very class for testing these methods, which mean hundreds of tests would fail all over the place :/